sources/packages/forms/src/form-definitions/programinformationrequest.json

```diff
@@ -1990,6 +1995,30 @@
           "customOptions": {},
           "useExactSearch": false,
           "id": "eiulpp"
+        },
+        {
+          "key": "institutionEnteredDetailsStudyperiodwarning",
+          "label": "Study period warning",
+          "input": false,
+          "tag": "p",
+          "attrs": [
+            {
+              "value": "",
+              "attr": ""
+            }
+          ],
+          "className": "alert alert-warning fa fa-exclamation-triangle w-100",
```

Comment:
It is weird to me the content that does not state the 42 days in the past.

Reply:
Discussed this with @hellolynn-tbtb .  As of now 42 days is not final, so the content here is kind of temporary and is subject to change when we get the final confirmation of days.